plugins/jira/requirements.txt

```diff
@@ -2,3 +2,4 @@
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
 jira==2.0.0
+timeout-decorator==0.5.0
```

Comment:
That does not seem to be used in this pr?

Reply:
> That does not seem to be used in this pr?

It's used in unit test, test failing without it.